automatic/ffmpeg-full/ffmpeg-full.nuspec

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>ffmpeg-full</id>
+    <version>4.4</version>
+    <title>FFmpeg</title>
+    <owners>chocolatey-community</owners>
```

Comment:
you'll need to add yourself as package owner

Reply:
Simply my Github Username? So like:
`<owners>chocolatey-community,VuiMuich</owners>`